apt/private/package_index.bzl

```diff
@@ -3,25 +3,36 @@
 load(":util.bzl", "util")
 
 def _fetch_package_index(rctx, url, dist, comp, arch, integrity):
-    target_triple = "{dist}/{comp}/{arch}".format(dist = dist, comp = comp, arch = arch)
+    # Split the URL by the '://' delimiter
+    protocol, rest = url.split("://")
+
+    # Split the rest of the URL by the '/' delimiter and take the first part
+    domain = rest.split("/")[0]
+
+    target_triple = "{domain}/{dist}/{comp}/{arch}".format(domain = domain, dist = dist, comp = comp, arch = arch)
```

Comment:
We needed to add `domain` to the `target_triple` because dist/comp/arch was duplicated across repositories since Ubuntu and Nvidia's repositories were both `focal/main/x86_64`.

`target_triple` is probably no longer the right term to use. I can change it as you see fit.